core/src/com/jmrapp/terralegion/game/world/chunk/gen/HillGenerator.java

```diff
@@ -32,6 +35,10 @@ public void generate(SimplexNoise noise, Chunk chunk, int x, int y) {
 		chunk.setBlock(BlockType.GRASS, x, finalY, false);
 		chunk.setWall(BlockType.DIRT_WALL, x, finalY, false);
 
+		if(this.grassRandom.nextInt(5) == 1){
```

Comment:
Using `this` doesn't really match with the project's code style. Also, where are the spaces? `{` should be preceded by a space.

Sorry if I seem nitpicky, but keeping a consistent code style is the only way to keep the code readable.